plugins/modules/discovery_workflow_manager.py

```diff
@@ -64,65 +65,137 @@
+                        type: str
+                    password:
+                        description: The password for CLI authentication, which is mandatory when using CLI credential.
+                        type: str
+                    enable_password:
+                        description: The enable password for CLI authentication, which is mandatory when using CLI credential.
+                        type: str
+            http_read_credential:
+                description: HTTP read credentials for hosting a device
+                type: dict
+                suboptions:
+                    username:
+                        description: The username for HTTP(S) authentication, which is mandatory when using HTTP credentials.
+                        type: str
+                    password:
+                        description: The password for HTTP(S) authentication. Mandatory for utilizing HTTP credentials.
+                        type: str
+                    port:
+                        description: The HTTP(S) port number, which is mandatory for using HTTP credentials.
+                        type: int
+                    secure:
+                        description: This is a flag for HTTP(S). Its usage is not mandatory for HTTP credentials.
+                        type: bool
+            http_write_credential:
+                description: HTTP write credentials for hosting a device
+                type: dict
+                suboptions:
+                    username:
+                        description: The username for HTTP(S) authentication, which is mandatory when using HTTP credentials.
+                        type: str
+                    password:
+                        description: The password for HTTP(S) authentication. Mandatory for utilizing HTTP credentials.
+                        type: str
+                    port:
+                        description: The HTTP(S) port number, which is mandatory for using HTTP credentials.
+                        type: int
+                    secure:
+                        description: This is a flag for HTTP(S). Its usage is not mandatory for HTTP credentials.
+                        type: bool
+            snmp_v2_read_credential:
+                description:
+                    - The SNMP v2 credentials to be created and used for contacting a device via SNMP protocol in read mode.
+                    - SNMP v2 also delivers data encryptions, but it uses data types.
+                type: dict
+                suboptions:
+                    desc:
+                        description: Name/Description of the SNMP read credential to be used for creation of snmp_v2_read_credential.
+                        type: str
+                    community:
+                        description: The read-only community string enables Cisco Catalyst Center to extract read-only data from device.
```

Comment:
what is read-only data? This may confuse. Can you check and update?

            Read-only community string for extracting data from devices.

Reply:
Removed only